app/bundles/ReportBundle/Model/ReportModel.php

```diff
@@ -716,4 +701,24 @@ private function getOperatorOptions(array $data)
 
         return $options;
     }
+
+    private function exportCSV($formatter,$report, $reportData, $handle, $page){
```

Comment:
This line should have a couple more spaces `private function exportCSV($formatter, $report, $reportData, $handle, $page) {`